tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -196,6 +196,24 @@ For a complete list of built-in GLSL functions see the :ref:`Shading Language do
 | vec_type **fwidth** ( vec_type )                                       |                                                  |
 +------------------------------------------------------------------------+--------------------------------------------------+
 
+**textureSize() workaround**
+
+OpenGL ES2.0 do not support ``TextureSize()``. If you need to get the size of a texture you need to do it in the old 
+fashioned way which is passing in a uniform with the texture size yourself.
+
+.. code-block:: glsl
+
+    // in shader
+    uniform sampler2D textureName;
+    uniform vec2 textureName_size;
+
+::
+
+    #in GDScript
+    material-name.set_shader_param("textureName", myTexture)
+    material-name.set_shader_param("textureName_size", size_of_myTexture)
```

Comment:
should be `material_name` instead of `material-name`.
gdscript does not support `-` for variable name.